lib/core/cmd.js

```diff
@@ -5,7 +5,7 @@ const { spawn } = require('child_process');
 module.exports = {
   exec: async (command, options={}) => {
     return new Promise((resolve, reject) => {
-      const items = command.split(' ');
+      const items = command.split(' '); //TODO problem, it is splitting even posix compliant path strings such as /Application\ Support/
```

Comment:
the os library returns something like .../Application Support/... on macos platform.

to be compliant with posix we could either translate it to .../Application\ Support/... or to .../ApplicationSupport/...

this command.split force us to this solution: .../ApplicationSupport/...